common/sampling.h

```diff
@@ -14,6 +14,7 @@ typedef struct llama_sampling_params {
     int32_t n_probs           = 0;     // if greater than 0, output the probabilities of top n_probs tokens.
     int32_t top_k             = 40;    // <= 0 to use vocab size
     float   top_p             = 0.95f; // 1.0 = disabled
+    float   min_p             = 1.0f; // 1.0 (or 0.0) = disabled
```

Comment:
You probably want this 0 to disable the sampler by default and keep current behavior? Lower requirements for getting merged if it's opt-in compared to trying to replace top-p by default.

Reply:
Agh I missed that one. Meant to standardize them all to 0.0f